src/celeritas/em/interactor/detail/PhysicsConstants.hh

```diff
@@ -64,6 +64,12 @@ CELER_CONSTEXPR_FUNCTION units::MevEnergy seltzer_berger_limit()
     return units::MevEnergy{1e3};  //! 1 GeV
 }
 
+//! Energy threshold between muon ionization models. TODO: make configurable
+CELER_CONSTEXPR_FUNCTION units::MevEnergy mu_ionization_limit()
```

Comment:
I initially thought `mu_bethe_block_limit` too, but worried since it wasn't consistent with our naming for the limit between the bremsstrahlung models it might be confusing (`mu_bethe_block_limit` would be the lower limit for the Bethe-Bloch model, but `seltzer_berger_limit` is the upper limit for SB). I agree `mu_ionization_limit` is misleading though... maybe we could rename `seltzer_berger_limit` to `rel_brems_limit`, or add `upper` and `lower` to the names or something?

Reply:
Adding low/upper definitely clarifies what the limit is referring to (of course, with a trade off of using the longer name).